--- tcm-2.20/src/gl/Makefile~	2005-11-22 23:16:14.847807032 +0100
+++ tcm-2.20/src/gl/Makefile	2005-11-22 23:29:01.824208896 +0100
@@ -35,7 +35,7 @@
 	version.o
 
 text2ps.o:
-	$(Cc) $(SYSFLAGS) -c text2ps.c
+	$(Cc) $(OPTFLAGS) $(SYSFLAGS) -c text2ps.c
 
 text2ps$(EXEEXTENSION): text2ps.o
 	$(Cc) text2ps.o -o $@ 
--- tcm-2.20/src/gl/text2ps.c~	2003-01-21 09:30:52.000000000 +0100
+++ tcm-2.20/src/gl/text2ps.c	2005-11-22 23:29:23.732878272 +0100
@@ -1,9 +1,7 @@
 #include <stdio.h>
 #include <time.h>
 
-#if defined(__CYGWIN__)
 #include <errno.h>
-#endif
 
 /*
  * Except for the point size all units in this program are tenths of
